rust/compiler_builtins.rs

```diff
@@ -42,10 +42,6 @@ macro_rules! define_panicking_intrinsics(
     }
 );
 
-define_panicking_intrinsics!("non-inline stack probes should not be used", {
-    __rust_probestack,
-});
```

Comment:
Do we need to remove this? i.e. can we keep it as a safeguard?

Reply:
It's best to remove this so we know there're no calls generated by accident. Note that we will never need `__rust_probestack` in the future -- once LLVM bug is fixed we'll switch to inline stack probes.

Reply:
Removing this ensures that non-inline stack probes are never generated.